rmgpy/solver/base.pyx

```diff
@@ -181,8 +181,8 @@ cdef class ReactionSystem(DASx):
         return (self.__class__, (self.termination,))
     
      
-    cpdef initializeModel(self, list coreSpecies, list coreReactions, list edgeSpecies, list edgeReactions, list surfaceSpecies,
-                          list surfaceReactions, list pdepNetworks=None, atol=1e-16, rtol=1e-8, sensitivity=False, 
+    cpdef initializeModel(self, list coreSpecies, list coreReactions, list edgeSpecies, list edgeReactions, list surfaceSpecies=[],
```

Comment:
Although that may be because it's only called unsafely by tests that aren't testing it explicitly.